tests/typescript/quote-props/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`types.ts - {"quoteProps":"as-needed"} format 1`] = `
 ====================================options=====================================
-parsers: ["typescript"]
+parsers: ["typescript", "babel-ts"]
```

Comment:
These changes aren't needed. If a test has `typescript` in `parsers`, it's [automatically verified](https://github.com/prettier/prettier/blob/5d3c137e1bbaec6df50b49d8764ecc6f80977715/tests_config/run_spec.js#L208) against `babel-ts`.